packages/SideNavigation/SubMenu/SubMenu.jsx

```diff
@@ -57,6 +52,17 @@ class SubMenu extends React.Component {
 
   state = {
     subMenuHeight: 0,
+    active: undefined,
+  }
+
+  componentWillMount() {
```

Comment:
`componentWillMount()` is considered legacy in React and will be removed in v17.

Could you please refactor based on React's recommendations?
https://reactjs.org/docs/react-component.html#unsafe_componentwillmount